Package@swift-5.1.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
   name: "UXKit",
   platforms: [
-    .macOS(.v10_12), .iOS(.v10)
+    .macOS(.v10_13), .iOS(.v13), .tvOS(.v13)
```

Comment:
That is quite a bump :-) But I think I'm fine w/ that. I think I've actually bumped Xcode even further.